roles/collect/defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 # defaults file for collect
 collect_format: 'csv'
 collect_report_name: 'hccm-openshift-usage-lookback'
-collect_endpoint_params: '?name={{ collect_report_name }}&format={{ collect_format }}'
```

Comment:
I was planning on moving collect_endpoint_params to be a collect task. This is because there is a new API param for the Operator Metering report API. It now requires `&namespace=` where the namespace specified would be the namespace that operator metering is. Currently that namespace is stored in the config that the collect role pulls in so I figured we could leverage that config to fill in the namespace param

Reply:
Makes sense. I've moved it.